hack/bootstrap-kubestellar.sh

```diff
@@ -0,0 +1,263 @@
+        exit 0;;
+    (-*)
+        echo "$0: unknown flag" >&2 ; exit 1;
+        exit 1;;
+    (*)
+        echo "$0: unknown positional argument" >&2; exit 1;
+        exit 1;;
+    esac
+    shift
+done
+
+if [ "$kcp_version" == "" ]; then
+    kcp_version=$(kcp_get_latest_version)
+fi
+
+if [ "$kubestellar_version" == "" ]; then
+    kubestellar_version=$(kubestellar_get_latest_version)
+fi
+
+if [ "$os_type" == "" ]; then
+    os_type=$(get_os_type)
+fi
+
+if [ "$arch_type" == "" ]; then
+    arch_type=$(get_arch_type)
+fi
+
+if [ "$folder" == "" ]; then
+    folder="$PWD"
+fi
+
+# Ensure kcp is installed
+if [ "$(kcp_installed)" == "false" ]; then
+    if [ "$verbose" != "" ]; then
+        echo "Installing kcp..."
+    fi
+    bash <(curl -s https://raw.githubusercontent.com/kcp-dev/edge-mc/main/hack/install-kcp-with-plugins.sh) --version $kcp_version --os $os_type --arch $arch_type --folder  $folder/kcp --create-folder $verbose
+    if [[ ! ":$PATH:" == *":$(get_full_path $folder/kcp/bin):"* ]]; then
+        export PATH=$PATH:$(get_full_path $folder/kcp/bin)
+    fi
+fi
+
+# Ensure kcp is running
+if [ "$(kcp_running)" == "false" ]; then
+    if [ "$verbose" != "" ]; then
+        echo "Starting kcp..."
+    fi
+    if [ "$kcp_address" == "" ]; then
+        kcp start >& kcp_log.txt &
+    else
+        kcp start --bind-address $kcp_address >& kcp_log.txt &
+    fi
+    export KUBECONFIG="$(pwd)/.kcp/admin.kubeconfig"
+    sleep 5
+    until kubectl ws . &> /dev/null
+    do
+        sleep 1
+    done
+    sleep 10
+    echo 'Export KUBECONFIG: export KUBECONFIG="$(pwd)/.kcp/admin.kubeconfig"'
+fi
+
+# Ensure KubeStellar is installed
+if [ "$(kubestellar_installed)" == "false" ]; then
+    if [ "$verbose" != "" ]; then
+        echo "Installing KubeStellar..."
+    fi
+    bash <(curl -s https://raw.githubusercontent.com/kcp-dev/edge-mc/main/hack/install-kubestellar.sh) --version $kubestellar_version --os $os_type --arch $arch_type --folder  $folder/kcp-edge --create-folder $verbose
+    if [[ ! ":$PATH:" == *":$(get_full_path $folder/kubestellar/bin):"* ]]; then
+        export PATH=$PATH:$(get_full_path $folder/kubestellar/bin)
+    fi
+fi
+
+# Ensure KubeStellar is running
+if [ "$(kubestellar_running)" == "false" ]; then
+    if [ "$verbose" != "" ]; then
+        echo "Starting KubeStellar..."
+    fi
+    kubestellar start --user kit $verbose
+fi
+
+# Ensure imw
+if [ "$kubestellar_imw" != "" ]; then
+    if [ "$verbose" != "" ]; then
+        echo "Ensuring IMW root:$kubestellar_imw..."
+    fi
+    if ! kubectl ws root:$kubestellar_imw &> /dev/null
+    then
+        if [ "$verbose" != "" ]; then
+            kubectl ws root
+            kubectl ws create $kubestellar_imw
+        else
+            kubectl ws root > /dev/null
+            kubectl ws create $kubestellar_imw > /dev/null
+        fi
+    fi
+fi
+
+# Ensure wmw
+if [ "$kubestellar_wmw" != "" ]; then
```

Comment:
@MikeSpreitzer since the ensure wmw was not taking the full path I used the option to create it in root.
As of now, the quickstart does not use this option but manually show the creation of imw and wmw so I would postpone this issue down the road. It's not a stopper.